cli/script/command-executor.ts

```diff
@@ -1347,6 +1365,47 @@ function serializeConnectionInfo(accessKey: string, preserveAccessKeyOnLogout: b
     log(`\r\nSuccessfully logged-in. Your session file was written to ${chalk.cyan(configFilePath)}. You can run the ${chalk.cyan("code-push logout")} command at any time to delete this file and terminate your session.\r\n`);
 }
 
+function sessionList(command: cli.ISessionListCommand): Promise<void> {
+    throwForInvalidOutputFormat(command.format);
+
+    return sdk.getAccessKeys()
```

Comment:
Do we want to add a `getSessions` method to the management SDK instead of defining this behavior in the CLI?